src/components/encoding-pane/filter-pane.scss

```diff
@@ -0,0 +1,49 @@
+$radius: 3px;
+$pill-height: 22px;
+$text-color: #212121;
+
+.filter-shelf {
+  background-color: white;
+  border-radius: $radius;
+  border: 1px solid #ddd;
+  margin-bottom: 4px;
+  max-height: 150px;
+  overflow: scroll;
+
+  :global .fa {
+    &.fa-times {
+      padding: 2px 3px 1px;
+      border-radius: $radius;
+
+      &:hover {
+        color: #fff;
+        background-color: #999;
+      }
+    }
+  }
+}
+
+.placeholder {
```

Comment:
We should design a way to share common styles between encodings and filter shelves instead of  just copy and paste style for the similar things -- otherwise it is hard to maintain consistency.